api.planx.uk/cors.ts

```diff
@@ -0,0 +1,33 @@
+import cors, { type CorsOptions } from "cors";
+
+const checkAllowedOrigins: CorsOptions["origin"] = (origin, callback) => {
+  if (!origin || origin === "null") return callback(null, true);
```

Comment:
This line is the only change here - when making requests from a local HTML file, origin is set to the truthy `"null"`;